frontend/public/models/yaml-templates.ts

```diff
@@ -194,7 +194,7 @@ spec:
   .setIn(
     [referenceForModel(k8sModels.ChargebackReportModel), 'default'],
     `
-apiVersion: metering.openshift.io/v1alpha1
+apiVersion: metering.openshift.io/v1
```

Comment:
`ChargebkacReportModel` still refers to `v1alpha1`. We can't just change the template YAML without changing the model and switching over entirely.

Are there API changes from `v1alpha1` to `v1`?

Reply:
I also see `v1alpha`:

https://github.com/openshift/console/blob/master/frontend/public/models/index.ts#L47
..and..
https://github.com/openshift/console/blob/master/frontend/public/components/chargeback.tsx#L30

..but I'm not 100% sure what there version number changes impact

Reply:
I changed `v1alpha` to `v1` in the areas I mentioned above ^^, from a url perspective:
http://0.0.0.0:9000/k8s/ns/openshift-metering/metering.openshift.io~v1~Report
...works, but 
http://0.0.0.0:9000/k8s/ns/openshift-metering/metering.openshift.io~v1~ReportQuery
...results in a `404: Not Found`

http://0.0.0.0:9000/k8s/ns/openshift-metering/metering.openshift.io~v1alpha1~ReportQuery
...continues to work, but have to manually enter `v1alpha1` in the url

Reply:
Also, when did this change happen?  Does this need to be backported to release-4.2?

Reply:
Backported changes to a 4.2 cluster and it worked there as well. Will do a followup PR to merge into release-4.2 branch.